ffi/wgpu.h

```diff
@@ -451,10 +480,12 @@ typedef WGPUNonZeroU64 WGPUId_Surface;
 
 typedef WGPUId_Surface WGPUSurfaceId;
 
+typedef uint64_t WGPUBufferSize;
+
 typedef struct {
   WGPUBufferId buffer;
   WGPUBufferAddress offset;
-  WGPUBufferAddress size;
+  WGPUBufferSize size;
```

Comment:
unrelated to your PR, it's unfortunate that cbindgen ignored `WHOLE`. We'll need to define it or make sure cbindgen sees it

Reply:
I feel cbindgen config/setup in general needs a good working through, it mentions the instructions on how to build wgpu-remote :)